libraries/AP_Mission/AP_Mission_Commands.cpp

```diff
@@ -163,7 +163,45 @@ bool AP_Mission::start_command_camera(const AP_Mission::Mission_Command& cmd)
         return false;
 
     case MAV_CMD_IMAGE_START_CAPTURE:
-        camera->take_picture();
+        // currently we don't handle the parameter 4 i.e. sequence number
```

Comment:
Maybe this would work instead?

        // check if this is a single picture request
        if (cmd.content.image_start_capture.total_num_images == 1) {
            if (cmd.content.image_start_capture.instance == 0) {
                // take pictures for every backend
                camera->take_picture();
                return true;
            }
            camera->take_picture(cmd.content.image_start_capture.instance-1);
            return true;
        }
        // multiple picture request
        if (cmd.content.image_start_capture.total_num_images == 0) {
            // take pictures forever
            if (cmd.content.image_start_capture.instance == 0) {
                // take pictures for every backend
                camera->take_multiple_pictures(cmd.content.image_start_capture.interval_s*1000, -1);
                return true;
            }
            camera->take_multiple_pictures(cmd.content.image_start_capture.instance-1, cmd.content.image_start_capture.interval_s*1000, -1);
            return true;
        } else {
            if (cmd.content.image_start_capture.instance == 0) {
                // take pictures for every backend
                camera->take_multiple_pictures(cmd.content.image_start_capture.interval_s*1000, cmd.content.image_start_capture.total_num_images);
                return true;
            }
            camera->take_multiple_pictures(cmd.content.image_start_capture.instance-1, cmd.content.image_start_capture.interval_s*1000, cmd.content.image_start_capture.total_num_images);
            return true;
        }
        return false;

Reply:
Done after a little change